overrides/android-override.json

```diff
@@ -1110,6 +1110,9 @@
         },
         "dummyWebMessageListener": {
             "state": "disabled"
+        },
+        "externalAppIntentFlags": {
+            "state": "enabled"
```

Comment:
We don't need this flag. The flat should be a kill-switch, we'd only add it here to disable it